### PR TITLE
Prepare fuchsia script to coexist with v1 and v2 of fuchsia builders.

### PR DESCRIPTION
**Pull request by godofredoc**
Prepares fuchsia scripts to work correctly when both v1 and v2 of the fuchsia build are running in their respective environments.

Bug: https://github.com/flutter/flutter/issues/126461
Bug: https://github.com/flutter/flutter/issues/135189

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [X] I listed at least one issue that this PR fixes in the description above.
- [X] I added new tests to check the change I am making or feature I am adding, or the PR is [test-exempt]. See [testing the engine] for instructions on writing and running engine tests.
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] I signed the [CLA].
- [X] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/main/CONTRIBUTING.md#style
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by godofredoc** [comment]
@zanderso any concerns about this change?

File: ci/builders/linux_fuchsia.json [hunk 1]
```diff
@@ -164,6 +164,7 @@
                 "parameters": [
                     "--engine-version",
                     "${REVISION}",
```
Comment (zanderso) [inline ci/builders/linux_fuchsia.json:166]: The "Magic variables" section of the docs here https://github.com/flutter/engine/blob/main/ci/builders/README.md#magic-variables should explain that `${REVISION}` is the empty string in presubmit.

**Review by zanderso — APPROVED** [review]
One note about updating the docs, assuming I've understood the semantics here correctly. Otherwise I don't see any issues.

File: ci/builders/linux_fuchsia.json (hunk 1 above)
Comment (godofredoc) [reply ci/builders/linux_fuchsia.json:166]: You are correct, revision is empty for presubmit. I'll update the docs.

**Pull request merged**